webhooks-extension/src/components/WebhookDisplayTable/tests/WebhookTable.test.js

```diff
@@ -85,4 +85,16 @@ describe('with webhooks', () => {
     const { getByText } = renderWithRouter(<WebhookDisplayTable match={{}} selectedNamespace="default"/>);
     await waitForElement(() => getByText("No webhooks created under namespace 'default', click 'Add Webhook' button to add a new one."));
   });
+
+  it('display branch modal when table row clicked', async () => {
+    jest.spyOn(API, "getWebhooks").mockImplementation(() => Promise.resolve(webhooks));
+    jest.spyOn(API, "getPipelineRuns").mockImplementation(() => Promise.resolve([]));
+    const { getByText, getByTestId } = renderWithRouter(<WebhookDisplayTable match={{}} selectedNamespace="*"/>);
+
+    await waitForElement(() => getByText(/first-test-webhook/i));
+
+    fireEvent.click(getByText(/first-test-webhook/i));
+    expect(getByTestId("webhook-branches-modal")).not.toBeNull();
```

Comment:
can we test for the header or other content instead of testid?

Reply:
sure!